internals/utils/src/is-valid-name.ts

```diff
@@ -1,5 +1,5 @@
 // Soft validation to prevent invalid API calls
-export function isValidName(name: string = ''): boolean {
+export function isValidName(name = ''): boolean {
```

Comment:
linter caught this after it was moved. not sure why it missed it while in cli but fix is simple enough

Reply:
Realized why this happened. We have some overrides configured for the CLI package:

https://github.com/vercel/vercel/blob/66fe3db888ff1d736023ce7cb70e8cdd7ff69ed3/package.json#L118-L137

I'm indifferent but my preference would be to not copy these overrides over and actually start listening to the linter.